iosApp/PhongKMMIC/Sources/Presentation/ViewId/ViewId.swift

```diff
@@ -56,13 +65,16 @@ enum ViewId {
             return login.rawValue
         case let .home(home):
             return home.rawValue
+        case let .surveyDetail(surveyDetail):
+            return surveyDetail.rawValue
         }
     }
 }
 
 extension View {
 
     func accessibility(_ viewId: ViewId) -> some View {
-        accessibilityIdentifier(viewId())
+        accessibilityElement(children: .contain)
```

Comment:
Just curious, why do we need this change? 🙏

Reply:
I faced an issue that the identifier can't be set to subviews if we set the identifier for the parent view.

Reply:
Nice one, I faced the same issue on my IC 👍